ergo/platforms/metaculus/question/continuous.py

```diff
@@ -148,8 +148,20 @@ def community_dist(self) -> dist.HistogramDist:
         # TODO (#306): Unify distributions interface
         # TODO (#307): Account for values out of range in
         #   ContinuousQuestion.community_dist()
+        denormalized_max = float(
+            self.denormalize_samples(self.prediction_histogram[-1][0])
```

Comment:
Could just call `true_from_normalized_value`, perhaps renamed to `denormalize_value`. Same in the two places below.

Reply:
`true_from_normalized_value` seems to only exist on LogQuestion

(IIRC I made it to make it easier to implement `denormalize_samples` on LogQuestions)

Reply:
What do you think about adding a `denormalize_value` to every Metaculus question class and using that?